index.js

```diff
@@ -1064,11 +1064,15 @@ Command.prototype.padWidth = function() {
 Command.prototype.optionHelp = function() {
   var width = this.padWidth();
 
+  var columns = process.stdout.columns || 80;
+  var descriptionWidth = columns - width - 4;
+  var minWidth = 40;
+
   // Append the help information
   return this.options.map(function(option) {
-    return pad(option.flags, width) + '  ' + option.description +
+    return pad(option.flags, width) + '  ' + optionalWrap(option.description, descriptionWidth, width + 2, minWidth) +
       ((!option.negate && option.defaultValue !== undefined) ? ' (default: ' + JSON.stringify(option.defaultValue) + ')' : '');
```

Comment:
The default value should be added to the description for optionalWrap, rather than added afterwards. I suggest a temporary variable to help with readability.

e.g.
```
var desc = option.description +
       ((!option.negate && option.defaultValue !== undefined) ? ' (default: ' + JSON.stringify(option.defaultValue) + ')' : '')
```